### PR TITLE
feat(docs): Add sendchamp provider documentation

### PR DESCRIPTION
**Pull request by peoray**
### What change does this PR introduce?

Add documentation for Sendchamp provider

### CODE REVIEW

File: apps/web/package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "envsetup": "chmod +x ./env.sh && ./env.sh && mv env-config.js ./public/env-config.js",
     "start:static:build": "pnpm envsetup:docker && http-server build -p 4200 --proxy http://localhost:4200?",
     "start:docker": "pnpm build && pnpm start:static:build",
-    "start:dev": "pnpm start",
+    "start:dev": "cross-env NODE_OPTIONS=--max-old-space-size=4096 pnpm start",
```
Comment (unicodeveloper) [inline apps/web/package.json:15]: Any reason for updating this?

**Review by unicodeveloper — COMMENTED** [review]
Please can you point me to the PR implementation of sendchamp as an SMS provider?

**Comment by peoray** [comment]
@unicodeveloper https://github.com/novuhq/novu/pull/3656

File: apps/web/package.json (hunk 1 above)
Comment (peoray) [reply apps/web/package.json]: Needed it for getting the web app running, without it, I usually get a JS heap memory limit error. Forgot to remove it after testing. Updated the code, should be fine now

**Comment by unicodeveloper** [comment]
> @unicodeveloper #3656

Okay. Dropped a comment on the implementation PR

**Review by BiswaViraj — CHANGES_REQUESTED** [review]
@peoray , Please add sendchamp doc link to this [file](https://github.com/novuhq/novu/blob/next/docs/docs/channels/sms/index.md) so that it will be shown on the homepage of [sms](https://docs.novu.co/channels/sms/)

**Comment by peoray** [comment]
@BiswaViraj done :)

**Review by BiswaViraj — APPROVED** [review]
🚀

**Pull request merged**